### DISCUSSION TITLE
Image object works in Windows but not WASM

### DISCUSSION THREAD

**Discussion by n2-holden** [unoplatform/uno#11094] [Q&A]
I'm trying to load an image from a URL.  The following works in my Windows build, but not in WASM:

XAML:
Image Name="thumbnail" 

C#:
thumbnail.Source = new BitmapImage(new Uri(url));

**Comment by jeromelaban** [comment]
Thanks for the question!

In general, this type of error may come because of content restrictions in the browser. Can you take a look at the browser debugger "network" and output console to see if there's an error?
